touch/utils.go

```
@@ -255,6 +271,74 @@ func GetMergedPointCloud(ctx context.Context, positions []toggleswitch.Switch, s
 		}
 
 		pcsInWorld = append(pcsInWorld, pcInWorld)
+
+		if saveFilesToCaptureDir {
+			if traceID == "" {
+				logger.Warn("saveFilesToCaptureDir was true but no traceID found on context, refusing to save files to capture dir")
```

The sanding module currently silently skips writing to the capture directory if there is no passId (aka traceId)

Here, because I have a saveFilesToCaptureDir bool, I log a warning if the bool is true but no passId/traceId is present